results/proto/taskrun.proto

```diff
@@ -19,6 +19,8 @@ message TaskRun {
 
 	TaskRunSpec spec = 4;
 	TaskRunStatus status = 5;
+	int64 cluster_info_hash = 6;
```

Comment:
What is this field? Is this being used, or should we remove it?

Reply:
There is according to a to-do comment by Jason inside taskrun.proto, it says  a cluster-info-hash can be added later as a unique identifier for taskrun. If right now adding more params is not necessary, I would remove it.